app/views/patients/transcripts/index.haml

```diff
@@ -0,0 +1,3 @@
+- if @transcripts.present?
```

Comment:
You should add a title to the page and provide a message if the patient doesn't have any transcripts